history/cultures/alteraci.txt

```diff
@@ -1,51 +1,75 @@
 1.1.1 = {
```

Comment:
Lordaeron, elves, tol'vir and many more cultures are very behind.
You need a new tech design. This is too wrong.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/124384988-e1919200-dce4-11eb-8440-05cb527dffc0.png)
![image](https://user-images.githubusercontent.com/46384992/124384991-e5bdaf80-dce4-11eb-8242-739d14a8c176.png)

</details>